### ISSUE TITLE
Move documentation from piku.github.io back to the main repository

### ISSUE THREAD

**Issue by jfmatth** [piku/piku#363]
For the documentation, we should put all documentation into the piku.github.io repo, no?   Seems redundant to keep it in two places.

I think we just need to remove the local links in the README and make sure everything is covered in the other repo.

Thoughts?

**Comment by rcarmo** [comment]
Yeah, I didn't want to remove everything immediately because people who check out the repo locally should get _something_. Not sure what would be most appropriate to keep without reviewing it all again.

**Comment by jfmatth** [comment]
Maybe point to the samples, give a quick example and then "To see more, goto ..."?

Just trying to be DRY like Django taught me :O

**Comment by chr15m** [comment]
With Rui's feedback on https://github.com/piku/piku/discussions/370 I think we should make the goal here be to have a) the doc sources in the main repo so it can be consistently updated in PRs that make changes to the code b) build the docs into the new documentation site as they are now. I've updated the issue title to reflect this.

**Comment by chr15m** [comment]
@rcarmo I just want to double check you are ok with my proposal here:

- Move the docs sources back into the main piku repository (single source of truth).
- Build piku.github.io on push to the main repository using those sources.
- Tidying up:
  - Move `img` into `docs`.
  - Remove the `binary` folder experiment.
  - Move the remaining examples out.

If I am missing some reason why it is better to have the docs source in a separate repository let me know.

**Comment by chr15m** [comment]
I still mean to do this.

**Comment by jfmatth** [comment]
I think i can help here, not done much with actions and testing docs.  Let me see what i can do.

**Comment by rcarmo** [comment]
Hi! I didn't pay any attention to the docs lately, but I'm good with it. Just can't spare the bandwidth.
